### PR TITLE
sql schema setup and update

### PR DESCRIPTION
**Pull request by everesio**
## What was changed
SQL support to schema-setup and schema-update jobs was added  

## Why?
Allow automatic schema creation and update also for SQL databases (so far only cassandra)

## Checklist

1. Closes 

2. How was this tested:

Tested on k8s with postgresql DB created by zalando postgres operator 

3. Any docs updates needed?

### CODE REVIEW

**Comment by natdm** [comment]
Any chance we can get this merged?

**Comment by franklinkim** [comment]
Hi team, seems like duplicate of #281. Would love to see one of them merged

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /charts
 /Chart.lock
-/.idea
\ No newline at end of file
+/.idea
+/*.iml
```
Comment (gajus) [inline .gitignore:4]: should not be part of this PR

File: Chart.yaml [hunk 2]
```diff
@@ -22,7 +22,6 @@ dependencies:
   - name: cassandra
     version: "0.14.3"
     repository: https://charts.helm.sh/incubator
-    enabled: false
```
Comment (gajus) [inline Chart.yaml:25]: should not be part of this PR

File: .gitignore (hunk 1 above)
Comment (everesio) [reply .gitignore]: Removed from the PR.

File: Chart.yaml (hunk 2 above)
Comment (everesio) [reply Chart.yaml]: Removed from the PR.

**Comment by mindaugasrukas** [comment]
Closing as a duplicate of https://github.com/temporalio/helm-charts/pull/281
